.github/workflows/publish-image-ghcr.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish GHCR Image
+on:
+  release: 
+    types: [published]
```

Comment:
Can we add another rigger that executes this workflow on every pull request... but only pushes the repo when we make a release? I think it should be possible